scripts/lint-style.py

```diff
@@ -232,8 +265,12 @@ def output_message(path, line_nr, code, msg):
         # filename first, then line so that we can call "sort" on the output
         print(f"{path} : line {line_nr} : {code} : {msg}")
     else:
+        if code.startswith("ERR"):
+            msg_type = "error"
+        if code.startswith("WRN"):
+            msg_type = "warning"
```

Comment:
I think the success / fail condition might actually all come down to the exit code of the python script, but marking this as a warning is sensible anyway.

Reply:
That's my suspicion as well but I don't think I've tested it.

Reply:
Do we want `warning` or `info`?

Reply:
warning seems fine